useUrlParams.js

```diff
@@ -1,9 +1,17 @@
 const urlParamHook = require('./urlParamHook')
 
-const defaultGetPath = () => window.location.href.replace(window.location.origin, '')
+const defaultGetPath = () => {
+	// extract hash as a parameter
+	const originalUrl = window.location.href.replace(window.location.origin, '')
+	const [urlWithoutHash, hash] = originalUrl.split('#')
+	const urlHasParam = urlWithoutHash.includes('?')
+	const hashParameter = hash !== undefined ? (urlHasParam ? `&hash=${hash}` : `?hash=${hash}`) : ''
+	const urlWithHashAsParam = `${urlWithoutHash}${hashParameter}`
+	return urlWithHashAsParam
```

Comment:
this is significantly more complicated, but the end result is turning the hash into a query parameter:

```
tram-one.io#features => tram-one.io?hash=features
```

This allows rlite to pick up the rest of the processing and create the final route object.